nevergrad/optimization/optimizerlib.py

```diff
@@ -995,8 +998,11 @@ def __init__(
         num_workers: int = 1,
         base_optimizer: base.OptCls = CMA,
         scale: tp.Optional[float] = None,
+        pareto_front_extractor: tp.Optional[str] = None,
     ) -> None:
         super().__init__(parametrization, budget=budget, num_workers=num_workers)
+        if pareto_front_extractor is not None:
+            self.__pareto_front_extractor = pareto_front_extractor
```

Comment:
typically, I am not completely sure this will work because of the `__`

Reply:
__ --> _ done